HelpSource/Classes/Float.schelp

```diff
@@ -65,3 +90,67 @@ pi.asStringPrec(6)
 7.4.asStringPrec(5)
 7.4.asStringPrec(50)
 ::
+
+subsection::Using Floats as replacement for Integers
+
+In SuperCollider, Floats are 64-bit wide. Because  an link::Classes/Integer:: is only 32-bit, it can capture only whole numbers up to code::-2147483648 .. +2147483647::, which is code::2^31:: (one bit is for the sign), or about code::2 x 10^9::.
```

Comment:
This still says (incorrectly) "one bit is for the sign". Also I think the phrasing "numbers in the range" is more natural than "numbers up to". I would recommend rewriting this as: "Because an Integer is 32-bit, it can only capture integers in the range -2147483648 .. +2147483647, or about `2 x 10^9`."

Reply:
ok, done.